tests/InputFiles/Poisson/ProductOfSinusoids1D.yaml

```diff
@@ -2,10 +2,16 @@
 # See LICENSE.txt for details.
 
 # Executable: SolvePoissonProductOfSinusoids1D
-# Check: parse;execute
+# Check: parse;execute;execute_check_output
```

Comment:
Do you think input files should have both `execute` and `execute_check_output` checks, or just one of them? I'm undecided.

Reply:
That's a reasonable point -- in a certain sense they're slightly different tests; One verifies that the program runs without erroring, and the other verifies aspects of its output. I think `execute_check_output` will fail anytime the `execute` would fail, so it's probably a better general policy to only have the `execute_check_output` if it is used, to reduce slightly the number and duration of tests, especially because the output-checking tests tend to be longer so that they can run to the point of producing possibly interesting output.